consul/default.toml

```diff
@@ -9,7 +9,7 @@ expect = "3"
 [server]
 bind = "127.0.0.1"
 data-dir = "/hab/svc/consul/data/consul"
-datacenter = "east-aws"
+datacenter = "dc1"
```

Comment:
What will happen if people use this plan, it updates automatically, and they use the default config here? will the consul cluster survive this? ;)

Reply:
This is more of a sample for usage rather than prescriptive. From talking with others who use this, along with my corporate configs, users provide custom settings here. Also, the datacenter setting is more of a label, helping to keep traffic tied to other consul nodes operating under that "datacenter".